lib/handlers/on-copy.js

```diff
@@ -289,7 +289,7 @@ async function copyHandler(server, messageHandler, connection, mailbox, update,
                                                     key => prepared.mimeTree.attachmentMap[key]
                                                 );
 
-                                                messageHandler.attachmentStorage.deleteMany(attachmentIds, maildata.magic, () => {});
+                                                messageHandler.attachmentStorage.deleteManyAsync(attachmentIds, maildata.magic);
```

Comment:
Add `.catch()` as well, otherwise if the Promise throws, then nothing will catch it.